initialization/initialization_test.go

```diff
@@ -63,11 +73,10 @@ func TestInitialize(t *testing.T) {
 func TestInitialize_Repeated(t *testing.T) {
 	r := require.New(t)
 
+	cfg, opts := getTestConfig(t)
 	init, err := NewInitializer(cfg, opts, id)
 	r.NoError(err)
-	if *log {
-		init.SetLogger(smlog.AppLog)
-	}
+	init.SetLogger(testLogger{})
```

Comment:
Shouldn't it be part of a `TestMain()` or `init()`?

Reply:
No, `init` is a new instance during every test and must not be re-used for the next test otherwise we are sharing state between tests again that shouldn't be shared.